modules/serverless-kafka-broker-configmap.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-configmap_{context}"]
+= Configuring Kafka broker settings
+
+You can configure the number of topic partitions, replication factor, and bootstrap servers for a Kafka broker, by creating a config map and referencing this config map in the Kafka `Broker` object.
+
+:FeatureName: Kafka broker
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have installed the OpenShift (`oc`) CLI.
```

Comment:
```suggestion
* You installed the OpenShift (`oc`) CLI.
```
Simpler tense.

Reply:
Wouldn't this then be inconsistent with other parts of the docs, e.g. above where it reads "You have created..."? I feel like "you installed" just reads strangely and wouldn't be consistent with the prereqs in all our other modules. Random example from our docs: https://docs.openshift.com/container-platform/4.11/serverless/develop/serverless-listing-event-sources.html#serverless-list-source-types-kn_serverless-listing-event-sources

I think this and the previous suggestions re wording would need to be raised as a suggestion for our docs as a whole, rather than changed for this one module, since they apply to a whole bunch of places. If there's anywhere in the guidelines where these are suggested and you could share links to those it would be helpful for future planning though.

Reply:
There isn't consistency in the docs themselves:
- https://docs.openshift.com/container-platform/4.11/machine_management/user_infra/adding-aws-compute-user-infra.html#prerequisites_adding-aws-compute-user-infra
- https://docs.openshift.com/container-platform/4.11/updating/updating-clusters-sro.html#updating-running-a-preflight-check-for-sro_updating-clusters-sro
- https://docs.openshift.com/container-platform/4.11/registry/configuring_registry_storage/configuring-registry-storage-baremetal.html#registry-configuring-registry-storage-rhodf-cephrgw_configuring-registry-storage-baremetal

ISG is explicit about using simple grammar whenever possible. :shrug:

Reply:
SSG, meanwhile, offers the simplest possible present tense alternative:

> JDK 11 or later is installed.

https://redhat-documentation.github.io/supplementary-style-guide/#prerequisites

Reply:
> There isn't consistency in the docs themselves
I specifically meant the serverless docs, since that's the product I work on :)
Thanks for the info!